src/pages/OrganizerAddModal.tsx

```diff
@@ -0,0 +1,283 @@
+import { Alert, AlertVariants } from '@/ui/Alert';
+import { FormElement } from '@/ui/FormElement';
+import { Inline } from '@/ui/Inline';
+import { Input } from '@/ui/Input';
+import { Modal, ModalSizes, ModalVariants } from '@/ui/Modal';
+import { RadioButtonGroup } from '@/ui/RadioButtonGroup';
+import { Stack } from '@/ui/Stack';
+import { Text, TextVariants } from '@/ui/Text';
+import { getValueFromTheme } from '@/ui/theme';
+import { Title } from '@/ui/Title';
+
+import { City, CityPicker } from './CityPicker';
+
+export const getValue = getValueFromTheme('organizerAddModal');
+
+const schema = yup
+  .object({
+    url: yup.string().required(),
+    name: yup.string().required(),
+    address: yup
+      .object({
+        streetAndNumber: yup.string().required(),
+        country: yup.string().oneOf(['BE', 'NL']).required(),
+        city: yup
+          .object({
+            label: yup.string().required(),
+            name: yup.string().required(),
+            zip: yup.string().required(),
+          })
+          .required(),
+      })
+      .required(),
+  })
+  .required();
+
+type FormData = yup.InferType<typeof schema>;
+
+const defaultValues: FormData = {
+  url: 'https://',
+  name: '',
+  address: {
+    country: 'BE',
+    streetAndNumber: '',
+    city: undefined,
+  },
+};
+
+type Props = {
+  prefillName: string;
+  visible: boolean;
+  onConfirm: (data: FormData) => Promise<void>;
+  onClose: () => void;
+};
+
+const OrganizerAddModal = ({
+  visible,
+  prefillName,
+  onConfirm,
+  onClose,
+}: Props) => {
+  const { t } = useTranslation();
+
+  const [urlInputComponent] = useAutoFocus<HTMLInputElement>({
+    retriggerOn: visible,
+  });
+
+  const {
+    register,
+    handleSubmit,
+    formState,
+    control,
+    reset,
+    watch,
+    setValue,
+    setError,
+  } = useForm<FormData>({
+    resolver: yupResolver(schema),
+    defaultValues,
+  });
+
+  const urlRegisterProps = register('url');
+
+  const watchedUrl = watch('url');
+  const watchedCountry = watch('address.country');
+
+  const getOrganizersByWebsiteQuery = useGetOrganizersByWebsiteQuery(
+    {
+      website: watchedUrl,
+    },
+    { enabled: !formState.errors.url },
+  );
+
+  const isUrlUnique =
+    // @ts-expect-error
+    getOrganizersByWebsiteQuery.data?.totalItems === 0;
+
+  const countries = useMemo(
+    () => [
+      {
+        label: t('countries.BE'),
+        value: 'BE',
+      },
+      {
+        label: t('countries.NL'),
+        value: 'NL',
+      },
+      // TODO: Add "Locatie in overleg met school"
+    ],
+    [t],
+  );
+
+  useEffect(() => {
+    setValue('name', prefillName);
+  }, [prefillName, setValue]);
+
+  const validateUniquenessUrl = async () => {
+    if (isUrlUnique) return true;
+    setError('url', { type: 'not_unique' });
+    return false;
+  };
+
+  const handleConfirm = async () => {
+    const isUniqueUrl = await validateUniquenessUrl();
+    if (!isUniqueUrl) return;
```

Comment:
I'm a bit lost with the `isUniqueUrl`.
But can probably a bit simpler? 
In this case the above suggestion can also be ignored.

```suggestion
    if (!isUrlUnique) {
    setError('url', { type: 'not_unique' });
    return
    };
```